test/metabase/driver/postgres_test.clj

```diff
@@ -7,9 +7,7 @@
              [query-processor-test :refer [rows]]
              [sync :as sync]
              [util :as u]]
-            [metabase.driver
-             [generic-sql :as sql]
-             postgres]
```

Comment:
Yeah I would just try to review all the changes in your code when you're wrapping up a PR and look for things like that and fix as needed. `cljr-clean-ns` is still better than doing things yourself since now it's a manual review of an automatic process instead of a manual review of a manual process